tests/test_document.py

```diff
@@ -27,17 +26,14 @@ def test_create(self):
         self.assertEqual(doc.id, test_other_id)
         self.assertEqual(doc.value, {'_id': test_other_id})
 
-
-    def test_no_id(self):
-        doc = Document({'someattr': 1})
-        self.assertRaises(AttributeError, getattr(doc), 'id')
-        self.assertRaises(AttributeError, getattr(doc), 'rev')
-
+    # def test_no_id(self):
```

Comment:
Why is this commented out?

Reply:
It gives error with flake8:
`F811 redefinition of unused 'test_no_id' from line 29`
I.e. the function id defined twice and unused.

Reply:
Can you rename the second test_no_id to something like test_id_len or what fits, it does not test a "no id" situation but whether a non-zero id is created (or more precisely, len 10 or longer).

Reply:
FYI: all methods starting with `test_` are called by pytest, so this "unused" is a bit misleading, the method is not called in other places explicitly

Reply:
Thanks, done

Reply:
Think I fixed it, please check.

Reply:
Looks good!